.github/workflows/ci.yml

```diff
@@ -116,7 +98,7 @@ jobs:
           rm targets.tar
 
       - name: Common sbt setup
-        run: $SBT ++${{ matrix.scala }} transferCommonResources transferPublishAndTagResources
+        run: $SBT transferCommonResources transferPublishAndTagResources
 
       - run: ./scripts/commonSetup
 
```

Comment:
From what I gather here, we will now merge the PR and then a Github action will notice the merge and push a version bump based on the PR labels. 

So now I wonder if we should restrict `publishAndTag` to commits generated by Github actions.

Reply:
Ah I see no longer use our own scripts to publish